packages/augur-ui/src/modules/account/components/stats.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import { LinearPropertyLabel } from "modules/common-elements/labels";
 import { formatNumber } from "utils/format-number";
 
-import Styles from "modules/account/components/stats.styles";
+import Styles from "modules/account/components/stats.styles.less";
```

Comment:
is there a plan to remove `.less` in the future? I'm ok with adding them temporary but I'd like to know how we are going to get rid of them.